components/salsa-macros/src/options.rs

```diff
@@ -433,3 +455,82 @@ impl<A: AllowedOptions> syn::parse::Parse for Options<A> {
         Ok(options)
     }
 }
+impl<A: AllowedOptions> quote::ToTokens for Options<A> {
+    fn to_tokens(&self, tokens: &mut proc_macro2::TokenStream) {
```

Comment:
I haven't read the diff to closely but what's this used for (should it replace some existing code?)